tests/containers/workbenches/jupyterlab/jupyterlab_test.py

```diff
@@ -20,7 +23,9 @@ class TestJupyterLabImage:
     APP_ROOT_HOME = "/opt/app-root/src"
 
     @allure.issue("RHOAIENG-11156")
-    @allure.description("Check that the HTML for the spinner is contained in the initial page.")
+    @allure.description(
+        "Check that the HTML for the spinner is contained in the initial page."
+    )
```

Comment:
your line length limit is a bit too low, imo; this would be a lot better to have everything on single line